docs/guides/getting-started.md

```diff
@@ -19,15 +22,16 @@ terraform {
 }
 provider "iterative" {}
 resource "iterative_task" "task" {
-  name  = "example"
-  cloud = "aws" # or any of: gcp, az, k8s
-  workdir {
-    input = "${path.root}/shared"
-  }
+  name   = "example"
+  cloud  = "aws" # or any of: gcp, az, k8s
```

Comment:
I would include a note telling that its going to create a default medium size machine and give an insight of what native type is going to be created just to be sure that the machine can be provisioned

Reply:
Link to a table with [generic machine types](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/generic-machine-types) and make default values explicit?

Reply:
- why did you move `workdir` further down?
- after this `hcl` block I'd suggest a sentence "See [the reference](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task) for a full list of options." or similar

Reply:
> _why did you move workdir further down?_

⚔️ Because the [style conventions](https://www.terraform.io/language/syntax/style#style-conventions) state the following:

> * _When both arguments and blocks appear together inside a block body, place all of the arguments together at the top and then place nested blocks below them. Use one blank line to separate the arguments from the blocks._
> * _Use empty lines to separate logical groups of arguments within a block._

Reply:
> _after this hcl block I'd suggest a sentence "See [the reference](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task) for a full list of options." or similar_

Great idea! If you deem https://github.com/iterative/terraform-provider-iterative/pull/375/commits/80f791ebdae9d1aa7dd5b4c418b303dd1cda2e89 too specific, feel free to update it with your suggestion.

Reply:
> Because the official style conventions

Upholding the spirit of said conventions in our context rather than taking them blindly as absolute truth, it makes more sense to specify `CWD` before `CMD`.

> update it with your suggestion

Done in 789c256.

Reply:
> _Upholding the spirit of said conventions in our context rather than taking them blindly as absolute truth, it makes more sense to specify `CWD` before `CMD`._

🤷🏼‍♂️ As you wish. Once you know the rules and still disagree, you're free to break them. Still, I wouldn't fight those guidelines in the same way I don't fight automated code formatters.

> _Done in https://github.com/iterative/terraform-provider-iterative/commit/789c2563f197002aa222a3fd2ccb1eb21a7a564b_

🙏🏼